### PR TITLE
DDRNet train recipe

### PR DESCRIPTION
**Pull request by lkdci**
Improved recipe for DDRNet training.
Results:
| model | mIoU | previous %mIoU diff |
| ------ | ----- | --------------- |
| DDRNet23 | 80.26 | 1.61 |
| DDRNet23-slim | 78.01| 1.41 |

Differences:
* Fixed backbone pretrained weights loading. Only weight for context path in DDRNet were loaded with ImageNet weights while the ImageNet ckpt includes also weights for spatial path and context-spatial interchanged layers.
* Different LR for encoder and decoder.
* Dice + CE-Edge combo loss instead of Hard mining loss.

### CODE REVIEW

File: src/super_gradients/recipes/cityscapes_stdc_base.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # STDC Base training params
 
 defaults:
-  - training_hyperparams: cityscapes_stdc_train_params
+  - training_hyperparams: cityscapes_train_params
```
Comment (shaydeci) [inline src/super_gradients/recipes/cityscapes_stdc_base.yaml:4]: Lets rename them to cityscapes_default_train_params

**Review by shaydeci — CHANGES_REQUESTED** [review]
Very Nice! just a tiny comment and then LGTM.

File: src/super_gradients/recipes/cityscapes_stdc_base.yaml (hunk 1 above)
Comment (lkdci) [reply src/super_gradients/recipes/cityscapes_stdc_base.yaml]: Done

**Comment by lkdci** [comment]
Addressed.

**Review by shaydeci — APPROVED** [review]
LGTM

**Pull request merged**